esp/scm/ws_dfu.ecm

```diff
@@ -707,34 +707,46 @@ ESPresponse [exceptions_inline, nil_remove, http_encode(0)] DFUGetFileMetaDataRe
     int64 TotalResultRows;
 };
 
-
 ESPrequest ListHistoryRequest
 {
-    string name;
+    string Name;
+};
+
+ESPStruct History
+{
+    string Name;
+    string Operation;
+    string Timestamp;
+    string IP;
+    string Path;
+    string Owner;
+    string Workunit;
 };
 
 ESPresponse [exceptions_inline, nil_remove] ListHistoryResponse
 {
-    binary xmlmap;
+    [depr_ver("1.36")] binary xmlmap;
+    ESParray<ESPStruct History> FileHistory;
```

Comment:
@AttilaVamos based on the offline conversation, let's define the File history array as follows:

`ESParray<ESPStruct History, Origin> History;`

notice we're now referencing the ESPStruct 'History' explicitely. Also, make sure the command line uses the same format.